how-to/integrate-with-sso/integrate-with-okta/package.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "openfin-workspace--integrate-with-okta",
+	"version": "10.3.0",
```

Comment:
Unfortunately as main is now based on workspace 11 you will need to sync this branch with main (there may be package.json conflicts as version numbers were bumped). You will also need to update the version number of the package.json file and the dependencies so that they use workspace 11 instead of 10 (if you look at the auth0 example on the main branch you will see the new version numbers).

Reply:
I merged from origin/main into my branch. Just want to check with you tomorrow morning to ensure that the process completed correctly.